lnwallet/size.go

```diff
@@ -0,0 +1,107 @@
+package lnwallet
+
+import (
+	"github.com/roasbeef/btcd/blockchain"
+)
+
+const (
+	WitnessFactor      = blockchain.WitnessScaleFactor
+	MaxTransactionCost = blockchain.MaxBlockCost / 10
+
+	// The weight(cost), which is different from the !size! (see BIP-141),
+	// is calculated as:
+	// Weight = 4 * BaseSize + WitnessSize (weight).
+	// BaseSize - size of the transaction without witness data (bytes).
+	// WitnessSize - witness size (bytes).
+	// Weight - the metric for determining the cost of the transaction.
+
+	// P2WSH:
+	//	- OP_0: 1 byte
+	//	- OP_DATA: 1 byte (WitnessScriptSHA256 lenght)
+	//	- WitnessScriptSHA256: 32 bytes
+	P2WSHSize = 1 + 1 + 32
```

Comment:
Nice job on the comments in this file! I modifying each definition to also include the total summed size of the variable would be a nice addition. It conveys a bit more information at a scan.